src/components/share/ShareDirective.js

```diff
@@ -47,6 +47,19 @@ goog.require('ga_permalink');
         }
         // Use clipboard API to copy URL in OS clipboard
         element.on('click', function() {
+
+            // the copy link for the iframe-preview is refreshed
+            // according to the preview changes
+            var previewFrame;
+            [previewFrame] = angular.element(document).find('iframe');
```

Comment:
Why use the find on the `document`? We should use the find on the element itself. Then we assure than only the iframe which is a child of this element, is selcted.

`element.find('iframe')` should be enought (or `angular.element(element).find('iframe')`